### ISSUE TITLE
[Suggestion] Click -> Click instead of Hold -> Release for Logic Link Tool

### ISSUE THREAD

**Issue by Kaiser-Indrasil** [tsunamayo/Starship-EVO#2703]
Just a quick one but very much a Quality of Life change.
I find myself dragging and dropping logic links over large distances, having to constantly hold my left mouse button.
It's a bit tedious and glitchy, since my mouse releases the hold for a split second sometimes, resulting in losing the link.
I've replaced the mouse recently but the issue is not gone (maybe it's Unity-related?)
Anyway, this change would surely remedy that, and relieve some tension in my hand.

**Comment by Vicomt** [comment]
I would totally love to see this as well.... linking systems from the bridge down to the front of the ship (500m away) is  painful.

**Comment by ProPeach** [comment]
Absolutely, as well as Shift + Click to connect multiple inputs from one output, a bit like Shift + Clicking folders in File Explorer

**Comment by ZachZent** [comment]
Related
https://github.com/tsunamayo/Starship-EVO/issues/2220
